src/backend/distributed/planner/fast_path_router_planner.c

```diff
@@ -53,6 +54,7 @@
 #include "distributed/shardinterval_utils.h"
 
 bool EnableFastPathRouterPlanner = true;
+bool EnableFastPathLocalExecutor = true;
```

Comment:
maybe we can name this as EnableLocalFastPathQueryOptimization to align with GUC name